extensions/deepcast/package.json

```diff
@@ -23,6 +23,14 @@
       "title": "API key",
       "description": "Your DeepL API key",
       "placeholder": "Enter your free DeepL API key"
+    },
+    {
+      "name": "pro",
+      "type": "checkbox",
```

Comment:
Hey! Looks like you've missed a `label` field here.
> Required, checkboxes only: The label of the checkbox. Shown next to the checkbox.

Check [the docs for preferences](https://developers.raycast.com/information/manifest#preference-properties) 😉

Reply:
Missed that, sry!